drivers/tty/serdev/pibridge.c

```diff
@@ -375,16 +373,17 @@ int pibridge_req_io(u8 addr, u8 cmd, u8 *snd_buf, u16 snd_len, u8 *rcv_buf, u16
 
 	if (pibridge_req_send_io(addr, cmd, snd_buf, snd_len)) {
 		pb_err(25, "send message error in io-req(addr:%d, cmd:%d, len:%d)\n",
-				addr, cmd, snd_len);
+		       addr, cmd, snd_len);
 		return -EIO;
 	}
 
-	if (sizeof(pkthdr) != pibridge_recv((u8 *)&pkthdr, sizeof(pkthdr))) {
+	if (sizeof(pkthdr) !=
+	    pibridge_recv((u8 *) &pkthdr, sizeof(pkthdr))) {
```

Comment:
I'd swap the arguments, i.e. call `pibridge_recv()` and compare the result to the expected value `sizeof(pkthdr)`. I think inverting the arguments as is done here is frowned upon in the kernel community. This was likely inadvertently carried over from piControl code.

Reply:
Agreed